packages/shared/src/types/props/shared.ts

```diff
@@ -140,3 +140,13 @@ export const imageSchema = z.object({
     mainUrl: z.string(),
     altText: z.string(),
 });
+
+export interface BylineWithImage {
+    author: string;
```

Comment:
The designs also have a role/description under the name, how about:
```
export interface BylineWithImage {
    name: string;
    description?: string;
    headshot: Image;
```

Reply:
Done